### DISCUSSION TITLE
why is rbd-data-pool-name flag mandatory in create-external-cluster-resources.py script?

### DISCUSSION THREAD

**Discussion by der-ali** [rook/rook#11390] [Q&A]
Hallo,
i want only to create Cephfs volumes, so i am wondering why must i create a rbd data pool to use the create-external-cluster-resources.py script?

**Comment by travisn** [comment]
@parth-gr Is there a way to not require the rbd data pool?

**Reply by parth-gr** [reply]
Currently, it would require to change a lot so it won't consider as mandatory, it is actually implemented with the downstream flow,

But we can remove this check, as UI would already check this. And can make it un-compulsory.
@travisn Thoughts

**Reply by subhamkrai** [reply]
I think we should not change the script based on what is on the UI but we can make changes to script that at least one fo the pool should be passed as agrument.

**Reply by parth-gr** [reply]
That can be also done,
But need to check with downstream and update the docs accordingly

**Reply by parth-gr** [reply]
@der-ali Having an option for choosing between would be a great idea.

Can you open an issue for the same, If you are interested in contributing towards it, that would be great.
